docs/quick-start.md

```diff
@@ -68,14 +68,29 @@ spec:
 
 ```bash
 # This command will create a namespace in the local cluster
-$ vela env init prod --namespace prod
+vela env init prod --namespace prod
+```
+
+The expected output:
+
+<details>
+
+```console
 environment prod with namespace prod created
```

Comment:
```suggestion
<details>
<summary>Expected output</summary>

```console
environment prod with namespace prod created
```

What about using `summary` to show `The expected output:` as title instead of `Details`?

Reply:
wow, great idea!